source/css/scss-altinn/generic/_variables.scss

```diff
@@ -5,7 +5,8 @@
 // Colors
 $blue-lighter: #E3F7FF;
 $blue-light: #CFF0FF;
-$blue: #1EAEF7;
+$blue: #008FD6; // Because of contrast, the blue primary color changed from #1EAEF7 to #008FD6 in January 2021. 
+$oldblue: #1EAEF7; // Old primary blue should still be used on modal backgrounds and buttons with black text. 
```

Comment:
if this comment is true, I expected a change in source/css/scss-common/objects/_modal.scss for the class `html.a-modal-background`?

Reply:
Thanks! .a-modal-background is now $oldblue